src/main/java/gyro/google/compute/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@DocGroup("Compute")
+package gyro.google.compute;
+
+import gyro.core.resource.DocGroup;
```

Comment:
This isn't related to PR.
This should be part of a different PR stating the need to add this.

Reply:
Sure I'll remove this.